api/registry.py

```diff
@@ -246,7 +246,7 @@ def _decode_data_url(cls, url):
             raise ValueError(
                 "Unsupported media type in data: URL: %s" % media_type
             )
-        html = base64.b64decode(encoded)
+        html = base64.b64decode(encoded.encode("utf-8")).decode("utf-8")
```

Comment:
This seems excessive?

Reply:
`b64decode` now takes a byte string. I was trying to use the base64 from the util class but then other encoding/decoding in this file break. I think I got fewer errors using the normal base64 package and doing the above than using the unicode aware base64 class and changing other strings in this file.